packages/origin-ui-core/src/components/Organization/OrganizationRegistrationStepper.tsx

```diff
@@ -22,8 +22,8 @@ export const OrganizationRegistrationStepper = () => {
         switch (stepName) {
             case STEP_NAMES.REGISTER_ORGANIZATION:
                 return <PlatformOrganizationRegistrationForm nextStep={setActiveStep} />;
-            case STEP_NAMES.CONNECT_OR_CREATE:
-                return <IRECConnectOrRegisterStep nextStep={(step) => setActiveStep(step)} />;
+            // case STEP_NAMES.CONNECT_OR_CREATE:
+            //     return <IRECConnectOrRegisterStep nextStep={(step) => setActiveStep(step)} />;
```

Comment:
Remove comments from the code, we have git history for preserving code.

Reply:
Kept it this way, because want to ask if i should I delete this file? I don't see any usage of this.

Reply:
Sure, check if it is used. If it's not, feel free to delete it.